Add all the mobile app (beta) documentation

marcalcobe opened the pull request:
Creating and adding all the structure for the beta release documentation for the OpenProject Mobile App.

# Ticket
https://community.openproject.org/projects/mobile-app-spike/work_packages/58347

# What are you trying to accomplish?
Having a full mobile app documentation for the mobile app beta release.

# Merge checklist

- [ ] Added/updated tests
- [ ] Added/updated documentation in Lookbook (patterns, previews, etc)
- [ ] Tested major browsers (Chrome, Firefox, Edge, ...)

marcalcobe committed:
Add all the mobile app (beta) documentation

Creating and adding all the structure for the beta release documentation for the OpenProject Mobile App.

MayaBerd commented on docs/mobile-app-guide/README.md:
```diff
@@ -0,0 +1,100 @@
+  title: Mobile app guide (Beta)
+  priority: 972
+description: User guide for the OpenProject Mobile application.
+keywords: Mobile app guide
```
```suggestion
keywords: Mobile app guide, mobile, mobile application, iOS, android, app store, appstore, app-store, google play
```

MayaBerd commented on docs/mobile-app-guide/README.md:
```diff
@@ -0,0 +1,100 @@
+---
+# Mobile app guide (Beta)
+
+We are excited to announce the **Beta Release** of the **OpenProject Mobile app** — now available in the **App Store** and **Google Play** Store. This Beta version marks the beginning of our journey to bring **OpenProject** to your mobile devices, enabling you to stay connected and productive wherever you are.
```
```suggestion
We are excited to announce the **Beta Release** of the **OpenProject Mobile app** — now available in the **App Store** and **Google Play Store**. This Beta version marks the beginning of our journey to bring **OpenProject** to your mobile devices, enabling you to stay connected and productive wherever you are.
```

MayaBerd commented on docs/mobile-app-guide/README.md:
```diff
@@ -0,0 +1,100 @@
+We are excited to announce the **Beta Release** of the **OpenProject Mobile app** — now available in the **App Store** and **Google Play** Store. This Beta version marks the beginning of our journey to bring **OpenProject** to your mobile devices, enabling you to stay connected and productive wherever you are.
+
+> [!NOTE]
+> The app is currently under active development, and core features are being continuously improved. We invite you to test the app, share feedback, and help shape the future of OpenProject Mobile.
```
Maybe we could also add a hint that both the app, AND this document are a work in progress? Something like: 

Hence, please also keep in mind that his page is a work in progress and will be updated along with the mobile app improvements. 
Last edited on: date, 2025.

marcalcobe replied:
Hi @MayaBerd aren't all documentations living elements? Do we do that for the web app documentation anywhere?

MayaBerd replied:
@marcalcobe True. I've recently added this to the overview of the JIRA migration docs, to try and keep any questions on our own tool at bay.

MayaBerd commented on docs/mobile-app-guide/README.md:
```diff
@@ -0,0 +1,100 @@
+
+The app is still under development, and many core features are planned for future updates, including:
+
+*   **Deep-linking (including on-premises support):** Seamlessly open specific work packages, projects, or comments directly from links in emails, chats, or browser pages — whether you’re using the cloud or an on-premises instance.
```
Is there already a framework for support options? Is it tied to the app support levels? We should clarify this somewhere.

marcalcobe replied:
No, there are no supports levels related to the mobile app and for now there is no intention of making anything related to the app Enterprise as far as I know.

I don't understand what do we need to clarify here. Deep-linking has nothing to do with our support plans. Maybe we should remove the "support" from the sentence "on-premises support" to avoid confusion.

MayaBerd commented on docs/mobile-app-guide/README.md:
```diff
@@ -0,0 +1,100 @@
+As a Beta user, your feedback is invaluable. If you encounter issues or have suggestions, please let us know:&nbsp;
+
+*   Using the **flow to provide feedback** directly from the **app** in the user settings.
+![Entry point for the mobile app flow to provide feedback](mobile_app_feedback_flow_entrypoint.png)
```
```suggestion
![Entry point for the OpenProject mobile app flow to provide user feedback](mobile_app_feedback_flow_entrypoint.png)
```

MayaBerd commented on docs/mobile-app-guide/README.md:
```diff
@@ -0,0 +1,100 @@
+
+*   Using the **flow to provide feedback** directly from the **app** in the user settings.
+![Entry point for the mobile app flow to provide feedback](mobile_app_feedback_flow_entrypoint.png)
+*   Create a **feedback work package** directly in OpenProject **Community**.
```
```suggestion
*   Create a **feedback work package** directly in [OpenProject **Community** instance](https://community.openproject.org/) and assign it to *Stream Mobile App*.
```

MayaBerd commented on docs/mobile-app-guide/README.md:
```diff
@@ -0,0 +1,100 @@
+To access and use the **OpenProject Mobile App (Beta)**, you’ll need the following:
+
+*   **An active OpenProject account:** Either from an **OpenProject Cloud** workspace or an **OpenProject On-premises** installation with API access enabled.
+*   Having a **signed certificate** (**https** not http) on your instance to be able to log in.
```
```suggestion
*   Having a **signed certificate** (**https**, not http) on your instance to be able to log in.
```

MayaBerd commented on docs/mobile-app-guide/README.md:
```diff
@@ -0,0 +1,100 @@
+
+*   **An active OpenProject account:** Either from an **OpenProject Cloud** workspace or an **OpenProject On-premises** installation with API access enabled.
+*   Having a **signed certificate** (**https** not http) on your instance to be able to log in.
+*   **Minimum OpenProject version:** 17.0.0
```
```suggestion
*   **OpenProject version:** 17.0.0 or above
```

MayaBerd commented on docs/mobile-app-guide/README.md:
```diff
@@ -0,0 +1,100 @@
+![Applications setting to enable the built-in OAuth](mobile_app_oauth_authentication.png)
+*   **Network connection:** Internet access is required for syncing data with your OpenProject instance.
+> [!NOTE]
+> Some features, such as deep-linking and realtime push notifications, may depend on your organization’s configuration or will become available in future updates.
```
```suggestion
> Some features, such as deep-linking and real-time push notifications, may depend on your organization’s configuration or will become available in future updates.
```

MayaBerd commented on docs/mobile-app-guide/README.md:
```diff
@@ -0,0 +1,100 @@
+
+## Disclaimer
+
+> [!NOTE]
```
```suggestion
> [!IMPORTANT]
```

MayaBerd commented on docs/mobile-app-guide/core-features/README.md:
```diff
@@ -0,0 +1,41 @@
+  title: Mobile app core features
+  priority: 800
+description: The OpenProject Mobile App is designed to keep you connected to your projects wherever you are. While it doesn’t replace the full desktop experience, it provides access to the essential tools you need to stay productive on the go.
+keywords: Mobile app core features
```
```suggestion
keywords: Mobile app core features, mobile, mobile app, core, mobile core, core features
```

MayaBerd commented on docs/mobile-app-guide/core-features/README.md:
```diff
@@ -0,0 +1,41 @@
+| [Projects](projects) | Browse your projects, access project-specific details, and stay on top of deadlines and activities. |
+| [Time Tracking](time-tracking) | Track time spent on work packages, run timers in focus mode, and keep your reporting accurate and up to date. |
+| [Work Packages](work-packages)| Create, view, and edit work packages directly from the app. Work packages are organized for quick access and easy collaboration. |
+| [Notification Centre](notification-centre) | Receive updates about comments, mentions, and work package changes — ensuring you never miss important activity. |
```
```suggestion
| [Notification Centre](notification-centre) | Receive updates about comments, mentions, and work package changes — ensuring you never miss an important activity. |
```

MayaBerd commented on docs/mobile-app-guide/core-features/README.md:
```diff
@@ -0,0 +1,41 @@
+| [Notification Centre](notification-centre) | Receive updates about comments, mentions, and work package changes — ensuring you never miss important activity. |
+| [User Settings](user-settings) | Configure your account, manage app preferences, and tailor notifications to suit your workflow. |
+
+> [!NOTE]
```
```suggestion
> [!TIP]
```

MayaBerd commented on docs/mobile-app-guide/core-features/home-dashboard/README.md:
```diff
@@ -0,0 +1,87 @@
+---
+sidebar_navigation:
+  title: Home Dashboard
```
We call it "Home page" in the core app. I would vote for using same terms whenever possible. Could we call it the same for mobile app?

marcalcobe replied:
Hi @MayaBerd, the functionalities of the "Home page" of the desktop are not the same as our Home Dashboard. The dashboard in mobile is fully configurable and personal to the user, so I would still like to keep the current name.

MayaBerd replied:
@marcalcobe thanks for clarifying. Then we should add it to the Glossary.

MayaBerd commented on docs/mobile-app-guide/core-features/home-dashboard/README.md:
```diff
@@ -0,0 +1,87 @@
+*   Displays recent notifications, such as mentions, comments, or work package updates.
+*   Enables you to **react quickly** without leaving the dashboard.
+
+![Home dashboard notifications widget](mobile_app_homewidget_notifications.png)
```
```suggestion
![Home dashboard notifications widget in the OpenProject mobile app](mobile_app_homewidget_notifications.png)
```

MayaBerd commented on docs/mobile-app-guide/core-features/home-dashboard/README.md:
```diff
@@ -0,0 +1,87 @@
+*   Displays projects you’ve marked as favorites for **quick access**.
+*   Ideal for prioritizing high-importance projects.
+
+![Home dashboard favorite projects widget](mobile_app_homewidget_favoriteprojects.png)
```
```suggestion
![Home dashboard displaying "favorite projects" widget in the OpenProject mobile app](mobile_app_homewidget_favoriteprojects.png)
```

MayaBerd commented on docs/mobile-app-guide/core-features/home-dashboard/README.md:
```diff
@@ -0,0 +1,87 @@
+*   Shows the **currently running timer** or allows you to start a new focus timer.
+*   Helps track work time efficiently and accurately.
+
+![Home dashboard time tracker widget](mobile_app_homewidget_timetracker.png)
```
```suggestion
![Home dashboard displaying the "time tracker" widget in OpenProject mobile app](mobile_app_homewidget_timetracker.png)
```

MayaBerd commented on docs/mobile-app-guide/core-features/home-dashboard/README.md:
```diff
@@ -0,0 +1,87 @@
+*   Provides an overview of **time logged during the current week**.
+*   Helps you monitor your productivity and ensure accurate reporting.
+
+![Home dashboard week time tracking widget](mobile_app_homewidget_weektimetracking.png)
```
```suggestion
![Home dashboard displaying "week time tracking" widget in the OpenProject mobile app](mobile_app_homewidget_weektimetracking.png)
```

MayaBerd commented on docs/mobile-app-guide/core-features/home-dashboard/README.md:
```diff
@@ -0,0 +1,87 @@
+*   Shows a snapshot of the portfolios available.
+*   Useful for portfolio managers and team members tracking **high-level progress** across multiple projects.
+
+![Home dashboard portfolios widget](mobile_app_homewidget_portfolios.png)
```
```suggestion
![Home dashboard displaying "portfolios" widget in the OpenProject mobile app](mobile_app_homewidget_portfolios.png)
```

MayaBerd commented on docs/mobile-app-guide/core-features/home-dashboard/README.md:
```diff
@@ -0,0 +1,87 @@
+*   Lists all **work packages currently assigned to you**.
+*   Includes types and status indicators for quick task management.
+
+![Home dashboard work packages assigned to me widget](mobile_app_homewidget_assignedtome.png)
```
```suggestion
![Home dashboard displaying "work packages assigned to me" widget in the OpenProject mobile app](mobile_app_homewidget_assignedtome.png)
```

MayaBerd commented on docs/mobile-app-guide/core-features/home-dashboard/README.md:
```diff
@@ -0,0 +1,87 @@
+### **Assigned to Me**
+
+*   Lists all **work packages currently assigned to you**.
+*   Includes types and status indicators for quick task management.
```
```suggestion
*   Includes work package types and status indicators for quick task management.
```

MayaBerd commented on docs/mobile-app-guide/core-features/home-dashboard/README.md:
```diff
@@ -0,0 +1,87 @@
+*   Shows a list of **recently edited work packages**.
+*   Allows **fast navigation** back to items that are work in progress.
+
+![Home dashboard work packages recently edited widget](mobile_app_homewidget_recentlyedited.png)
```
```suggestion
![Home dashboard displaying "work packages recently edited" widget in the OpenProject mobile app](mobile_app_homewidget_recentlyedited.png)
```

MayaBerd commented on docs/mobile-app-guide/core-features/home-dashboard/README.md:
```diff
@@ -0,0 +1,87 @@
+
+## Customizing the Dashboard
+
+While the mobile dashboard is designed for simplicity, the **order and visibility of widgets** can be customized by using the edit widgets button on the top navigation bar, you are allowed to:
```
```suggestion
While the mobile dashboard is designed for simplicity, the **order and visibility of widgets** can be customized by using the **edit widgets button** on the right side of the top navigation bar. Clicking the icon allows you to:
```

MayaBerd commented on docs/mobile-app-guide/core-features/home-dashboard/README.md:
```diff
@@ -0,0 +1,87 @@
+*   Activate and deactivate widgets
+*   Reorder widgets
+
+![Home dashboard edit widgets entry point](mobile_app_home_addwidget.png)
```
```suggestion
![Home dashboard edit widgets entry point in OpenProject mobile app](mobile_app_home_addwidget.png)
```

MayaBerd commented on docs/mobile-app-guide/core-features/home-dashboard/README.md:
```diff
@@ -0,0 +1,87 @@
+*   Reorder widgets
+
+![Home dashboard edit widgets entry point](mobile_app_home_addwidget.png)
+![Home dashboard edit widgets bottom sheet configuration](mobile_app_home_addwidget_bottomsheet.png)
```
```suggestion
![Home dashboard edit widgets bottom sheet configuration in OpenProject mobile app](mobile_app_home_addwidget_bottomsheet.png)
```

MayaBerd commented on docs/mobile-app-guide/core-features/notification-centre/README.md:
```diff
@@ -0,0 +1,38 @@
+sidebar_navigation:
+  title: Notification Centre
+  priority: 750
+description: A centralized view of all updates, mentions, and activities that require the user’s attention.
```
```suggestion
description: A centralized view of all updates, mentions, and activities that require the user’s attention in OpenProject mobile app.
```

MayaBerd commented on docs/mobile-app-guide/core-features/notification-centre/README.md:
```diff
@@ -0,0 +1,38 @@
+  title: Notification Centre
+  priority: 750
+description: A centralized view of all updates, mentions, and activities that require the user’s attention.
+keywords: Mobile app features Notification Centre
```
```suggestion
keywords: Mobile app Notification Centre, mobile notification, mobile notifications, notification, mobile mention
```

marcalcobe committed:
Update docs/mobile-app-guide/README.md

Co-authored-by: Maya Berdygylyjova <MayaBerd@users.noreply.github.com>

marcalcobe committed:
Update docs/mobile-app-guide/README.md

Co-authored-by: Maya Berdygylyjova <MayaBerd@users.noreply.github.com>

marcalcobe committed:
Update docs/mobile-app-guide/README.md

Co-authored-by: Maya Berdygylyjova <MayaBerd@users.noreply.github.com>

marcalcobe committed:
Update docs/mobile-app-guide/README.md

Co-authored-by: Maya Berdygylyjova <MayaBerd@users.noreply.github.com>

marcalcobe committed:
Update docs/mobile-app-guide/README.md

Co-authored-by: Maya Berdygylyjova <MayaBerd@users.noreply.github.com>

marcalcobe committed:
Update docs/mobile-app-guide/README.md

Co-authored-by: Maya Berdygylyjova <MayaBerd@users.noreply.github.com>

marcalcobe committed:
Update docs/mobile-app-guide/README.md

Co-authored-by: Maya Berdygylyjova <MayaBerd@users.noreply.github.com>

marcalcobe committed:
Update docs/mobile-app-guide/README.md

Co-authored-by: Maya Berdygylyjova <MayaBerd@users.noreply.github.com>

marcalcobe committed:
Update docs/mobile-app-guide/core-features/README.md

Co-authored-by: Maya Berdygylyjova <MayaBerd@users.noreply.github.com>

marcalcobe committed:
Update docs/mobile-app-guide/core-features/README.md

Co-authored-by: Maya Berdygylyjova <MayaBerd@users.noreply.github.com>

marcalcobe committed:
Update docs/mobile-app-guide/core-features/README.md

Co-authored-by: Maya Berdygylyjova <MayaBerd@users.noreply.github.com>

marcalcobe committed:
Update docs/mobile-app-guide/core-features/home-dashboard/README.md

Co-authored-by: Maya Berdygylyjova <MayaBerd@users.noreply.github.com>

marcalcobe committed:
Update docs/mobile-app-guide/core-features/home-dashboard/README.md

Co-authored-by: Maya Berdygylyjova <MayaBerd@users.noreply.github.com>

marcalcobe committed:
Update docs/mobile-app-guide/core-features/home-dashboard/README.md

Co-authored-by: Maya Berdygylyjova <MayaBerd@users.noreply.github.com>

marcalcobe committed:
Update docs/mobile-app-guide/core-features/home-dashboard/README.md

Co-authored-by: Maya Berdygylyjova <MayaBerd@users.noreply.github.com>

marcalcobe committed:
Update docs/mobile-app-guide/core-features/home-dashboard/README.md

Co-authored-by: Maya Berdygylyjova <MayaBerd@users.noreply.github.com>

marcalcobe committed:
Update docs/mobile-app-guide/core-features/home-dashboard/README.md

Co-authored-by: Maya Berdygylyjova <MayaBerd@users.noreply.github.com>

marcalcobe committed:
Update docs/mobile-app-guide/core-features/home-dashboard/README.md

Co-authored-by: Maya Berdygylyjova <MayaBerd@users.noreply.github.com>

marcalcobe committed:
Update docs/mobile-app-guide/core-features/home-dashboard/README.md

Co-authored-by: Maya Berdygylyjova <MayaBerd@users.noreply.github.com>

marcalcobe committed:
Update docs/mobile-app-guide/core-features/home-dashboard/README.md

Co-authored-by: Maya Berdygylyjova <MayaBerd@users.noreply.github.com>

marcalcobe committed:
Update docs/mobile-app-guide/core-features/home-dashboard/README.md

Co-authored-by: Maya Berdygylyjova <MayaBerd@users.noreply.github.com>

marcalcobe committed:
Update docs/mobile-app-guide/core-features/home-dashboard/README.md

Co-authored-by: Maya Berdygylyjova <MayaBerd@users.noreply.github.com>

marcalcobe committed:
Update docs/mobile-app-guide/core-features/notification-centre/README.md

Co-authored-by: Maya Berdygylyjova <MayaBerd@users.noreply.github.com>

marcalcobe committed:
Update docs/mobile-app-guide/core-features/notification-centre/README.md

Co-authored-by: Maya Berdygylyjova <MayaBerd@users.noreply.github.com>

MayaBerd commented on docs/mobile-app-guide/core-features/notification-centre/README.md:
```diff
@@ -0,0 +1,38 @@
+
+## Receiving and Viewing Notifications
+
+After logging in to the app, you will automatically receive notifications generated in their OpenProject environment. Whenever an update occurs—such as a comment, status change, or mention—the notification will appear inside the Notification Centre. You can simply open the app and navigate to the Notification Centre to view any new, unread items waiting for their attention.
```
```suggestion
After logging into the app, you will automatically receive notifications generated in their OpenProject environment. Whenever an update occurs—such as a comment, status change, or mention—the notification will appear inside the Notification Centre. You can simply open the app and navigate to the Notification Centre to view any new, unread items waiting for their attention.
```

MayaBerd commented on docs/mobile-app-guide/core-features/notification-centre/README.md:
```diff
@@ -0,0 +1,38 @@
+
+You can also choose to display **all notifications**, not just unread ones. By toggling the switch to view all items, the Notification Centre expands to include both read and unread notifications. This is useful for revisiting past activity or reviewing a complete history of updates.
+
+![Mobile app Notification Centre](mobile_app_notification_centre.png)
```
```suggestion
![OpenProject mobile app Notification Centre](mobile_app_notification_centre.png)
```

MayaBerd commented on docs/mobile-app-guide/core-features/notification-centre/README.md:
```diff
@@ -0,0 +1,38 @@
+
+Tapping on a notification opens the associated work package directly in the mobile app. You are taken to the Activity tab of that work package, where the specific update that triggered the notification is highlighted. This makes it easy to understand the context of the notification and respond immediately if needed.
+
+![Mobile app work package activity tab](mobile_app_workpackage_activity.png)
```
```suggestion
![OpenProject mobile app work package activity tab](mobile_app_workpackage_activity.png)
```

MayaBerd commented on docs/mobile-app-guide/core-features/notification-centre/README.md:
```diff
@@ -0,0 +1,38 @@
+
+When a notification is opened or manually marked as read, it is removed from the **Unread** inbox. This allows you to maintain a clear and focused view of only pending items. For quicker inbox management, the Notification Centre also provides an option to **mark all notifications as read** from the top bar, clearing the unread list in a single action.
+
+![Mobile app mark a notification as read by swiping it](mobile_app_notification_centre_swipe_markasread.png)
```
```suggestion
![Mark a notification as read by swiping it in the OpenProject mobile app](mobile_app_notification_centre_swipe_markasread.png)
```

MayaBerd commented on docs/mobile-app-guide/core-features/notification-centre/README.md:
```diff
@@ -0,0 +1,38 @@
+When a notification is opened or manually marked as read, it is removed from the **Unread** inbox. This allows you to maintain a clear and focused view of only pending items. For quicker inbox management, the Notification Centre also provides an option to **mark all notifications as read** from the top bar, clearing the unread list in a single action.
+
+![Mobile app mark a notification as read by swiping it](mobile_app_notification_centre_swipe_markasread.png)
+![Mobile app mark a notification as read using the more menu](mobile_app_notification_centre_moreaction_markasread.png)
```
```suggestion
![Mark a notification as read using the more menu in the OpenProject mobile app](mobile_app_notification_centre_moreaction_markasread.png)
```

MayaBerd commented on docs/mobile-app-guide/core-features/notification-centre/README.md:
```diff
@@ -0,0 +1,38 @@
+
+## Switching Between Notification Queries
+
+The Notification Centre includes filters that allow you to switch between different notification queries, similar to the web interface. You can choose alternative inboxes—such as “Assigne,” “Mention,” or other notification categories their instance provides. When switching queries, the list updates immediately to show only the notifications relevant to that filter.
```
```suggestion
The Notification Centre includes filters that allow you to switch between different notification queries, similar to the web interface. You can choose alternative inboxes—such as “Assignee,” “Mention,” or other notification categories their instance provides. When switching queries, the list updates immediately to show only the notifications relevant to that filter.
```

MayaBerd commented on docs/mobile-app-guide/core-features/notification-centre/README.md:
```diff
@@ -0,0 +1,38 @@
+
+The Notification Centre includes filters that allow you to switch between different notification queries, similar to the web interface. You can choose alternative inboxes—such as “Assigne,” “Mention,” or other notification categories their instance provides. When switching queries, the list updates immediately to show only the notifications relevant to that filter.
+
+![Mobile app switch the Notification Centre query](mobile_app_notification_centre_queries.png)
```
```suggestion
![OpenProject mobile app switch the Notification Centre query](mobile_app_notification_centre_queries.png)
```

MayaBerd commented on docs/mobile-app-guide/core-features/projects/README.md:
```diff
@@ -0,0 +1,48 @@
+  title: Projects
+  priority: 780
+description: An organized view of all portfolios, programs, and projects you have access to.
+keywords: Mobile app features projects
```
```suggestion
keywords: Mobile app features projects, mobile project, mobile portfolio, mobile program, program, programme, project, portfolio, mobile app
```

MayaBerd commented on docs/mobile-app-guide/core-features/projects/README.md:
```diff
@@ -0,0 +1,48 @@
+*   **Browse a hierarchical list of all projects**, expandable to show programs and child projects.
+*   **Filter the list** to display only your **favorite projects** for quick access.
+
+![Mobile app project module index](mobile_app_projectsmodule.png)
```
```suggestion
![Project module index in the OpenProject mobile app](mobile_app_projectsmodule.png)
```

MayaBerd commented on docs/mobile-app-guide/core-features/projects/README.md:
```diff
@@ -0,0 +1,48 @@
+    *   Displays all **projects or programs that belong to the selected portfolio, program, or project**.
+    *   Helps you explore the hierarchy and drill down into individual initiatives.
+
+![Mobile app project details overview](mobile_app_project_overview.png)
```
```suggestion
![Project details overview in the OpenProject mobile app](mobile_app_project_overview.png)
```

MayaBerd commented on docs/mobile-app-guide/core-features/projects/README.md:
```diff
@@ -0,0 +1,48 @@
+    *   Helps you explore the hierarchy and drill down into individual initiatives.
+
+![Mobile app project details overview](mobile_app_project_overview.png)
+![Mobile app project details work packages](mobile_app_project_workpackages.png)
```
```suggestion
![Project details work packages in the OpenProject mobile app](mobile_app_project_workpackages.png)
```

MayaBerd commented on docs/mobile-app-guide/core-features/projects/README.md:
```diff
@@ -0,0 +1,48 @@
+
+![Mobile app project details overview](mobile_app_project_overview.png)
+![Mobile app project details work packages](mobile_app_project_workpackages.png)
+![Mobile app project details sub-elements in the project](mobile_app_project_inthisportfolio.png)
```
```suggestion
![Project details sub-elements in the project in the OpenProject mobile app](mobile_app_project_inthisportfolio.png)
```

MayaBerd commented on docs/mobile-app-guide/core-features/time-tracking/README.md:
```diff
@@ -0,0 +1,50 @@
+  title: Time tracking
+  priority: 770
+description: Enables you to easily record, review, and manage the time you spend on your work.
+keywords: Mobile app features time tracking
```
```suggestion
keywords: Mobile app features time tracking, time tracking, mobile time tracking, mobile time, log time, time logging, time-logging, time management, time, time entry, time log, timer focus, focus timer
```

MayaBerd commented on docs/mobile-app-guide/core-features/time-tracking/README.md:
```diff
@@ -0,0 +1,50 @@
+
+This view is ideal for maintaining accurate records and verifying past entries.
+
+![Mobile app time tracking week logged time index page](mobile_app_timetrackingmodule.png)
```
```suggestion
![Time tracking week logged time index page in the OpenProject mobile app](mobile_app_timetrackingmodule.png)
```

MayaBerd commented on docs/mobile-app-guide/core-features/time-tracking/README.md:
```diff
@@ -0,0 +1,50 @@
+This view is ideal for maintaining accurate records and verifying past entries.
+
+![Mobile app time tracking week logged time index page](mobile_app_timetrackingmodule.png)
+![Mobile app time tracking index page with a day of the week selected](mobile_app_timetrackingmodule_dayfocused.png)
```
```suggestion
![Time tracking index page with a day of the week selected in the OpenProject mobile app](mobile_app_timetrackingmodule_dayfocused.png)
```

MayaBerd commented on docs/mobile-app-guide/core-features/time-tracking/README.md:
```diff
@@ -0,0 +1,50 @@
+*   **Let the timer run in the background**, even if you switch apps or lock your device.
+*   **Stop the timer** when finished, and convert the session into a logged time entry.
+
+![Mobile app timer running in a focus mode](mobile_app_focustimer.png)
```
```suggestion
![Timer running in a focus mode in the OpenProject mobile app](mobile_app_focustimer.png)
```

MayaBerd commented on docs/mobile-app-guide/core-features/time-tracking/README.md:
```diff
@@ -0,0 +1,50 @@
+*   **Stop the timer** when finished, and convert the session into a logged time entry.
+
+![Mobile app timer running in a focus mode](mobile_app_focustimer.png)
+![Mobile app timer running in the background shown in the index page](mobile_app_timerrunning.png)
```
```suggestion
![Timer running in the background shown in the index page in the OpenProject mobile app](mobile_app_timerrunning.png)
```

MayaBerd commented on docs/mobile-app-guide/core-features/time-tracking/README.md:
```diff
@@ -0,0 +1,50 @@
+*   **Specify the date, duration, and activity type**.
+*   **Ensure accurate reporting** even if you didn’t use the focus timer.
+
+![Mobile app log time modal](mobile_app_logtime.png)
```
```suggestion
![Log time modal in the OpenProject mobile app](mobile_app_logtime.png)
```

MayaBerd commented on docs/mobile-app-guide/core-features/user-settings/README.md:
```diff
@@ -0,0 +1,33 @@
+  title: User settings
+  priority: 740
+description: Personalize their mobile app experience, configure notifications, manage account details, and provide feedback.
+keywords: Mobile app features user settings
```
```suggestion
keywords: Mobile app features user settings, user settings, user rights, user permissions, mobile permission, mobile settings, mobile user, mobile notification settings
```

MayaBerd commented on docs/mobile-app-guide/core-features/user-settings/README.md:
```diff
@@ -0,0 +1,33 @@
+The Notification Settings section allows you to configure how and when they receive notifications:
+*   **Deactivate Local Notifications**: You can disable OS-level local pull notifications, preventing alerts from appearing on their device.
+*   **Adjust Notification Preferences**: You can fine-tune which notifications they receive, such as participation alerts, non-participation alerts, or date-related reminders. Changes are applied to both mobile and desktop apps, and notifications will adhere to the new settings.
+
```
This section has no screenshots. Was this by design?

marcalcobe replied:
Yes it was by design, I don't think adding mockups here brings any value, the screens are only switch on and off.

MayaBerd commented on docs/mobile-app-guide/core-features/work-packages/README.md:
```diff
@@ -0,0 +1,72 @@
+  title: Work packages
+  priority: 760
+description: Browse, search, create, update, and collaborate on work packages directly from your mobile device.
+keywords: Mobile app features work packages
```
```suggestion
keywords: Mobile app features work packages, work package, work packages, create work package, edit work package, share work package, mobile app
```

MayaBerd commented on docs/mobile-app-guide/core-features/work-packages/README.md:
```diff
@@ -0,0 +1,72 @@
+
+In addition to filtering, you can quickly locate specific work packages through the search functionality. Tapping the search icon opens a search bar where users can type keywords and instantly view matching results.
+
+![Mobile app work packages index page](mobile_app_workpackages_index.png)
```
```suggestion
![Work packages index page in the OpenProject mobile app](mobile_app_workpackages_index.png)
```

MayaBerd commented on docs/mobile-app-guide/core-features/work-packages/README.md:
```diff
@@ -0,0 +1,72 @@
+
+Creating new work packages is also fully supported. You can tap the &quot;+&quot; button to begin the creation process. After selecting the work package type and project, you can fill in the required fields to quickly create a new entry. For more detailed entries, you may choose to open the full attribute list and complete optional fields before saving, resulting in a comprehensive work package w ...
+
+![Mobile app work package creation bottom sheet to select a type](mobile_app_create_workpackage_type.png)
```
```suggestion
![Work package creation bottom sheet to select a type in the OpenProject mobile app](mobile_app_create_workpackage_type.png)
```

MayaBerd commented on docs/mobile-app-guide/core-features/work-packages/README.md:
```diff
@@ -0,0 +1,72 @@
+Creating new work packages is also fully supported. You can tap the &quot;+&quot; button to begin the creation process. After selecting the work package type and project, you can fill in the required fields to quickly create a new entry. For more detailed entries, you may choose to open the full attribute list and complete optional fields before saving, resulting in a comprehensive work package w ...
+
+![Mobile app work package creation bottom sheet to select a type](mobile_app_create_workpackage_type.png)
+![Mobile app work package creation details modal](mobile_app_create_workpackage_details.png)
```
```suggestion
![Work package creation details modal in the OpenProject mobile app](mobile_app_create_workpackage_details.png)
```

MayaBerd commented on docs/mobile-app-guide/core-features/work-packages/README.md:
```diff
@@ -0,0 +1,72 @@
+
+You can open any work package to view its details and make updates. Editing is initiated through the pencil icon, giving access to fields such as the title, description, status, and any available custom fields. All updates are applied directly in the work package, so users can easily refine information as work progresses.
+
+![Mobile app work package details on the overview tab](mobile_app_workpackage_details_overview.png)
```
```suggestion
![Work package details on the overview tab in the OpenProject mobile app](mobile_app_workpackage_details_overview.png)
```

MayaBerd commented on docs/mobile-app-guide/core-features/work-packages/README.md:
```diff
@@ -0,0 +1,72 @@
+
+The module supports collaboration through the **Activity** tab, where you can participate in discussions. Comments can be added or replied to, mentions can be used to involve other users, and images may be included directly in the comment body. This makes it easier to share updates, coordinate with your team, or provide visual information on the go.
+
+![Mobile app work package details on the activity tab](mobile_app_workpackage_activity.png)
```
```suggestion
![Work package details on the activity tab in the OpenProject mobile app](mobile_app_workpackage_activity.png)
```

MayaBerd commented on docs/mobile-app-guide/core-features/work-packages/README.md:
```diff
@@ -0,0 +1,72 @@
+
+Attachments are handled through the **Files** tab. You can upload files, photos, or videos from their device, or take a new picture using the camera and attach it immediately. This allows work packages to hold all relevant documentation in one place.
+
+![Mobile app work package details on the files tab](mobile_app_workpackage_files.png)
```
```suggestion
![Work package details on the files tab in the OpenProject mobile app](mobile_app_workpackage_files.png)
```

MayaBerd commented on docs/mobile-app-guide/core-features/work-packages/README.md:
```diff
@@ -0,0 +1,72 @@
+
+Relations between work packages can be managed in the **Relations** tab, where you can connect tasks to one another by adding new relations when needed. Similarly, watchers can be added from the **Watchers** tab to keep stakeholders informed about progress or changes.
+
+![Mobile app work package details on the activity tab](mobile_app_workpackage_relations.png)
```
```suggestion
![Work package details on the activity tab in the OpenProject mobile app](mobile_app_workpackage_relations.png)
```

MayaBerd commented on docs/mobile-app-guide/core-features/work-packages/README.md:
```diff
@@ -0,0 +1,72 @@
+Relations between work packages can be managed in the **Relations** tab, where you can connect tasks to one another by adding new relations when needed. Similarly, watchers can be added from the **Watchers** tab to keep stakeholders informed about progress or changes.
+
+![Mobile app work package details on the activity tab](mobile_app_workpackage_relations.png)
+![Mobile app work package details on the activity tab](mobile_app_workpackage_watchers.png)
```
```suggestion
![Work package details on the activity tab in the OpenProject mobile app](mobile_app_workpackage_watchers.png)
```

MayaBerd commented on docs/mobile-app-guide/core-features/work-packages/README.md:
```diff
@@ -0,0 +1,72 @@
+
+Additionally, you can set reminders from within the **More** menu. By configuring a reminder for a specific date and time, the app will notify you when it’s time to return to that work package or take action on it.
+
+![Mobile app work package details with the more actions menu open](mobile_app_workpackage_moreactions.png)
```
```suggestion
![Work package details with the more actions menu open in the OpenProject mobile app](mobile_app_workpackage_moreactions.png)
```

MayaBerd commented on docs/mobile-app-guide/core-features/work-packages/README.md:
```diff
@@ -0,0 +1,72 @@
+
+Work packages can be easily shared using the device’s **native sharing features**. From within a work package, you can access the system share from the **More** menu to send a direct link to the item through messaging apps, email, collaboration tools, or any other app supported by their operating system. This simplifies cross-platform communication and makes it effortless to bring others into the ...
+
+![Mobile app work package details sharing bottom sheet open](mobile_app_workpackage_share.png)
```
```suggestion
![Work package details sharing bottom sheet open in the OpenProject mobile app](mobile_app_workpackage_share.png)
```

MayaBerd commented on docs/mobile-app-guide/faq/README.md:
```diff
@@ -0,0 +1,122 @@
+  title: Mobile app FAQ
+  priority: 800
+description: FAQ of the OpenProject Mobile app.
+keywords: Mobile app FAQ
```
```suggestion
keywords: Mobile app FAQ, faq, mobile app questions, OpenProject mobile app
```

MayaBerd commented on docs/mobile-app-guide/faq/README.md:
```diff
@@ -0,0 +1,122 @@
+
+## Why is the OpenProject Mobile App being released in a Beta state?
+
+The app is released in **Beta** to provide early access to users while the core functionality is still under development. This allows the OpenProject team to gather **feedback from real users**, identify issues, and make improvements before the full public release. The Beta release ensures that users can start using the app for essential tasks while the remaining advanced features and refinements ...
```
```suggestion
The app is released in **Beta** state to provide early access to users while the core functionality is still under development. This allows the OpenProject team to gather **feedback from real users**, identify issues, and make improvements before the full public release. The Beta release ensures that users can start using the app for essential tasks while the remaining advanced features and refinements are being implemented.
```

MayaBerd commented on docs/mobile-app-guide/faq/README.md:
```diff
@@ -0,0 +1,122 @@
+
+The app requires an **active internet connection** to sync data with your OpenProject instance.
+
+## What do I need to log in to the app?
```
Maybe "How can I log into the app?"

MayaBerd commented on docs/mobile-app-guide/faq/README.md:
```diff
@@ -0,0 +1,122 @@
+
+## What do I need to log in to the app?
+
+You can log in using your OpenProject Cloud account or an On-Premises instance, username and password. Please ensure:
```
```suggestion
You can log in using your OpenProject Cloud account or your username and password for your On-Premises instance. Please ensure:
```

MayaBerd commented on docs/mobile-app-guide/faq/README.md:
```diff
@@ -0,0 +1,122 @@
+*   **Portfolios**: Snapshot of portfolio-level progress.
+*   **Assigned to Me**: List of work packages assigned to you.
+*   **Recently Viewed**: Quick access to recently opened items.
+
```
```suggestion
[Learn more about Home Dashboard in the OpenProject mobile app](../core-features/home-dashboard/).
```

MayaBerd commented on docs/mobile-app-guide/faq/README.md:
```diff
@@ -0,0 +1,122 @@
+    *   **Overview tab**: Description, status, and attributes
+    *   **Work Packages tab**: List of associated tasks
+    *   **Child Projects/Programs tab**: Projects or programs under the selected item
+
```
```suggestion
[Learn more about Projects module in the OpenProject mobile app](../core-features/projects/).
```

MayaBerd commented on docs/mobile-app-guide/faq/README.md:
```diff
@@ -0,0 +1,122 @@
+*   Logging time or starting a timer for focused work
+*   Setting reminders
+*   **Sharing work packages** using the device’s native sharing options
+    
```
```suggestion
    [Learn more about Work Packages module in the OpenProject mobile app](../core-features/work-packages/).
```

MayaBerd commented on docs/mobile-app-guide/faq/README.md:
```diff
@@ -0,0 +1,122 @@
+*   **Time Entries Index**: View current and past weeks and their logged time entries.
+*   **Timer Focus Mode**: Track work in real time with a background timer.
+*   **Log Time**: Record time spent on specific work packages manually.
+
```
```suggestion
 [Learn more about Time Tracking in OpenProject mobile app](../core-features/time-tracking/).
```

MayaBerd commented on docs/mobile-app-guide/faq/README.md:
```diff
@@ -0,0 +1,122 @@
+*   Mark individual notifications or all notifications as read.
+*   Switch between notification queries to filter by participation, mentions, or other custom inboxes.
+*   Toggle between viewing only unread or all notifications.
+
```
```suggestion
 [Learn more about Notifications in OpenProject mobile app](../core-features/notification-centre/).
```

MayaBerd commented on docs/mobile-app-guide/faq/README.md:
```diff
@@ -0,0 +1,122 @@
+*   Update **personal details** (name, email, etc.)
+*   Configure **notification settings** and deactivate OS-level notifications
+*   Provide **feedback** directly to the OpenProject Community project
+
```
```suggestion
 [Learn more about configuring user settings in OpenProject mobile app](../core-features/user-settings/).
```

MayaBerd commented on docs/mobile-app-guide/faq/README.md:
```diff
@@ -0,0 +1,122 @@
+*   Configure **notification settings** and deactivate OS-level notifications
+*   Provide **feedback** directly to the OpenProject Community project
+
+## Are all OpenProject features available on mobile?
```
```suggestion
## Are all OpenProject features available in the mobile app?
```

MayaBerd commented on docs/mobile-app-guide/faq/README.md:
```diff
@@ -0,0 +1,122 @@
+
+## Are all OpenProject features available on mobile?
+
+No. The mobile app is a **companion app** and is currently in Beta. Some advanced features from the web or desktop version are not yet available, including:
```
```suggestion
No. The mobile app is a **companion app** and is currently in Beta status. Some advanced features of the web or desktop versions are not yet available, including:
```

MayaBerd commented on docs/mobile-app-guide/faq/README.md:
```diff
@@ -0,0 +1,122 @@
+*   Your instance meets the **minimum version** requirement (17.0.0), or the OAuth feature flag is enabled.
+*   Ensure your credentials are correct, and you have **API access** enabled on On-Premises instances.
+
+If none of the above solves the problem, check the [Login Troubleshooting Guide](https://www.openproject.org/docs/mobile-app-guide/first-steps/login-troubleshooting/).
```
```suggestion
If none of the above solves the problem, check the [Login Troubleshooting Guide](.../first-steps/login-troubleshooting/).
```

MayaBerd commented on docs/mobile-app-guide/core-features/notification-center/README.md:
```diff
@@ -0,0 +1,38 @@
+---
```
In the documentation for the native app we say Center, not Centre. Do you mind if we stick to that spelling?

MayaBerd commented on docs/mobile-app-guide/first-steps/README.md:
```diff
@@ -0,0 +1,65 @@
+  title: First Steps: Access, Download, and Log In
+  priority: 800
+description: Follow these steps to install and start using the OpenProject Mobile App (Beta).
+keywords: mobile app first steps
```
```suggestion
keywords: mobile app first steps, getting started, log in, log-in, login, mobile log in, openproject mobile app, download, access, install, mobile installation
```

MayaBerd commented on docs/mobile-app-guide/first-steps/README.md:
```diff
@@ -0,0 +1,65 @@
+Before downloading the app, please ensure your environment meets the following prerequisites:
+
+*   **An active OpenProject account:** Either from an **OpenProject Cloud** workspace or an **OpenProject On-premises** installation with API access enabled.
+*   Having a **signed certificate** (**https** not http) on your instance to be able to log in.
```
```suggestion
*   A **signed certificate** (**https** not http) on your instance to be able to log in.
```

MayaBerd commented on docs/mobile-app-guide/first-steps/README.md:
```diff
@@ -0,0 +1,65 @@
+    *   **iOS 15** or later
+    *   **Android 12** or later
+*   **Built-in OAuth applications enabled:** Make sure that the built-in OAuth applications are **enabled in your administration settings** (`{BASE_URL}/admin/oauth/applications`). This is required for successful login from the mobile app.
+![Applications setting to enable the built-in OAuth](mobile_app_oauth_authentication.png)
```
```suggestion
![Applications setting to enable the built-in OAuth in OpenProject](mobile_app_oauth_authentication.png)
```

MayaBerd commented on docs/mobile-app-guide/first-steps/README.md:
```diff
@@ -0,0 +1,65 @@
+![Applications setting to enable the built-in OAuth](mobile_app_oauth_authentication.png)
+*   **Network connection:** Internet access is required for syncing data with your OpenProject instance.
+> [!NOTE]
+> Some features, such as deep-linking and realtime push notifications, may depend on your organization’s configuration or will become available in future updates.
```
```suggestion
> Some features, such as deep-linking and real-time push notifications, may depend on your organization’s configuration or will become available in future updates.
```

MayaBerd commented on docs/mobile-app-guide/first-steps/README.md:
```diff
@@ -0,0 +1,65 @@
+## Open the App
+
+After installation, open the app on your device. You’ll be greeted with a **short onboarding** on the app features and then prompted to **log in** to your OpenProject instance.
+![Mobile app initial onboarding page](mobile-app-onboarding.png)
```
```suggestion
![Initial onboarding page of the OpenProject mobile app](mobile-app-onboarding.png)
```

MayaBerd commented on docs/mobile-app-guide/first-steps/README.md:
```diff
@@ -0,0 +1,65 @@
+*   Once your instance is confirmed, log in using your **OpenProject username and password** in the browser modal that opened.
+![Mobile app browser sign in screen](mobile-app-browser-sign-in.png)
+    
+*   The app will ask you permission to have full access to the **OpenProject API v3** so account and securely connect to your workspace.
```
```suggestion
*   The app will ask you for a permission to have full access to the **OpenProject API v3** to authorize your account and securely connect to your workspace.
```

MayaBerd commented on docs/mobile-app-guide/first-steps/README.md:
```diff
@@ -0,0 +1,65 @@
+![Mobile app browser sign in screen](mobile-app-browser-sign-in.png)
+    
+*   The app will ask you permission to have full access to the **OpenProject API v3** so account and securely connect to your workspace.
+![Mobile app browser API v3 authorisation screen](mobile-browser-oauth-authorization.png)
```
```suggestion
![OpenProject mobile app browser API v3 authorisation screen](mobile-browser-oauth-authorization.png)
```

MayaBerd commented on docs/mobile-app-guide/first-steps/login-troubleshooting/README.md:
```diff
@@ -0,0 +1,122 @@
+---
+sidebar_navigation:
+  title: Login Troubleshooting
```
```suggestion
  title: Mobile App Troubleshooting
```

MayaBerd replied:
I would call it more generically, Troubleshooting, so that later on this page can be expanded and other types of troubleshooting can be added. 

If you agree, then we should change Login Troubleshooting to H2.

marcalcobe replied:
This page wasn't intended to solve all types of troubleshooting of the app, but rather only those related to the login process. I would rather deal with other error cases and possible troubleshootings in a separate page.

MayaBerd commented on docs/mobile-app-guide/first-steps/login-troubleshooting/README.md:
```diff
@@ -0,0 +1,122 @@
+  title: Login Troubleshooting
+  priority: 700
+description: Resolve the most common issues logging in to the OpenProject Mobile app.
+keywords: Mobile App Login Troubleshooting
```
```suggestion
keywords: Mobile App Login Troubleshooting, log in error, mobile app troubleshooting, mobile app error
```

MayaBerd commented on docs/mobile-app-guide/first-steps/login-troubleshooting/README.md:
```diff
@@ -0,0 +1,122 @@
+
+# Login Troubleshooting
+
+If you’re having trouble logging in to the **OpenProject Mobile App**, the following sections will help you identify and resolve the most common issues.
```
```suggestion
If you’re having trouble logging into the **OpenProject Mobile App**, the following sections will help you identify and resolve the most common issues.
```

MayaBerd commented on docs/mobile-app-guide/first-steps/login-troubleshooting/README.md:
```diff
@@ -0,0 +1,122 @@
+You know that your instance is running not on the minimum supported version, OpenProject 17.0.0, and the login fails with a browser error such as _“An authorization error has occurred. The client is not authorised to perform this request using this method.”_.
+
+**Cause:**  
+The OpenProject Mobile App requires your instance to be on **OpenProject version 17.0.0 or later**.  
```
```suggestion
The OpenProject Mobile App requires your instance to be on **OpenProject version 17.0.0 or higher**.  
```

MayaBerd commented on docs/mobile-app-guide/first-steps/login-troubleshooting/README.md:
```diff
@@ -0,0 +1,122 @@
+**Cause:**  
+The URL you entered may be incorrect, inaccessible, or not using HTTPS.
+
+**How to Fix:**
```
Maybe consider **Solution** instead of **How to Fix** ?

MayaBerd commented:
@marcalcobe thank you very much for putting this guide together! I've learned a lot about the mobile app while reviewing it. I will probably like to do another review round once all links are added, or more changes are merged. 

A side note: if you still have to original screenshots for the troubleshooting section, could we change the color of the highlighting boxes from red to green, to keep it in the same style as the rest of the docs (except for really old ones)?

marcalcobe committed:
Update docs/mobile-app-guide/core-features/work-packages/README.md

Co-authored-by: Maya Berdygylyjova <MayaBerd@users.noreply.github.com>

marcalcobe committed:
Update docs/mobile-app-guide/core-features/work-packages/README.md

Co-authored-by: Maya Berdygylyjova <MayaBerd@users.noreply.github.com>

marcalcobe committed:
Update docs/mobile-app-guide/faq/README.md

Co-authored-by: Maya Berdygylyjova <MayaBerd@users.noreply.github.com>

marcalcobe committed:
Update docs/mobile-app-guide/faq/README.md

Co-authored-by: Maya Berdygylyjova <MayaBerd@users.noreply.github.com>

marcalcobe committed:
Update docs/mobile-app-guide/faq/README.md

Co-authored-by: Maya Berdygylyjova <MayaBerd@users.noreply.github.com>

marcalcobe committed:
Update docs/mobile-app-guide/faq/README.md

Co-authored-by: Maya Berdygylyjova <MayaBerd@users.noreply.github.com>

marcalcobe committed:
Update docs/mobile-app-guide/faq/README.md

Co-authored-by: Maya Berdygylyjova <MayaBerd@users.noreply.github.com>

marcalcobe committed:
Update docs/mobile-app-guide/faq/README.md

Co-authored-by: Maya Berdygylyjova <MayaBerd@users.noreply.github.com>

marcalcobe committed:
Update docs/mobile-app-guide/faq/README.md

Co-authored-by: Maya Berdygylyjova <MayaBerd@users.noreply.github.com>

marcalcobe committed:
Update docs/mobile-app-guide/faq/README.md

Co-authored-by: Maya Berdygylyjova <MayaBerd@users.noreply.github.com>

marcalcobe committed:
Update docs/mobile-app-guide/faq/README.md

Co-authored-by: Maya Berdygylyjova <MayaBerd@users.noreply.github.com>

marcalcobe committed:
Update docs/mobile-app-guide/faq/README.md

Co-authored-by: Maya Berdygylyjova <MayaBerd@users.noreply.github.com>

marcalcobe committed:
Update docs/mobile-app-guide/faq/README.md

Co-authored-by: Maya Berdygylyjova <MayaBerd@users.noreply.github.com>

marcalcobe committed:
Update docs/mobile-app-guide/faq/README.md

Co-authored-by: Maya Berdygylyjova <MayaBerd@users.noreply.github.com>

marcalcobe committed:
Update docs/mobile-app-guide/first-steps/README.md

Co-authored-by: Maya Berdygylyjova <MayaBerd@users.noreply.github.com>

marcalcobe committed:
Update docs/mobile-app-guide/first-steps/README.md

Co-authored-by: Maya Berdygylyjova <MayaBerd@users.noreply.github.com>

marcalcobe committed:
Update docs/mobile-app-guide/first-steps/README.md

Co-authored-by: Maya Berdygylyjova <MayaBerd@users.noreply.github.com>

marcalcobe committed:
Update docs/mobile-app-guide/first-steps/README.md

Co-authored-by: Maya Berdygylyjova <MayaBerd@users.noreply.github.com>

marcalcobe committed:
Update docs/mobile-app-guide/first-steps/README.md

Co-authored-by: Maya Berdygylyjova <MayaBerd@users.noreply.github.com>

marcalcobe committed:
Update docs/mobile-app-guide/first-steps/README.md

Co-authored-by: Maya Berdygylyjova <MayaBerd@users.noreply.github.com>

marcalcobe committed:
Update docs/mobile-app-guide/first-steps/login-troubleshooting/README.md

Co-authored-by: Maya Berdygylyjova <MayaBerd@users.noreply.github.com>

marcalcobe committed:
Update docs/mobile-app-guide/first-steps/login-troubleshooting/README.md

Co-authored-by: Maya Berdygylyjova <MayaBerd@users.noreply.github.com>

marcalcobe committed:
Update docs/mobile-app-guide/first-steps/login-troubleshooting/README.md

Co-authored-by: Maya Berdygylyjova <MayaBerd@users.noreply.github.com>

MayaBerd committed:
Merge branch 'release/17.0' into mobille-app-documentation

MayaBerd committed:
Apply suggestions from code review

marcalcobe committed:
Update docs/mobile-app-guide/core-features/notification-centre/README.md

Co-authored-by: Maya Berdygylyjova <MayaBerd@users.noreply.github.com>

marcalcobe committed:
Changes on the mobile documentation after marketing review

All remaining changes after @MayaBerd review.

marcalcobe commented:
Hi @MayaBerd, I've added all the remaining changes after your previous review. Also, we included the links to the OS Stores that for now don't work because they are not yet public, but once the app is published they will be the same urls so it will work then.

@psatyal, @oleksii-openproject (or @yanzubrytskyi) it would be great if you could also take a look into this documentation from a copywriting and technical side.

marcalcobe committed:
Fix on documentation links

The links between documentation pages where broken, modified and fixed all of them

marcalcobe committed:
Fix error with two dots on page title
